### DISCUSSION TITLE
[Question]: Is max_disappeared relative to camera FPS or the detect.fps setting?

### DISCUSSION THREAD

**Discussion by edalquist** [blakeblackshear/frigate#13336] [Ask A Question] [answered]
### What is your question?

When setting detect.max_disappeared is the frame count relative to the detect.fps or the camera FPS?

With their default values of max_disappeared=25 and fps=5 it implies 5 seconds of an object missing means it is gone. But my cameras all run at 15fps, so is it actually just under 2 seconds?

**Comment by hawkeye217** [comment, marked as answer]
Because `max_disappeared` is operating on your decoded `detect` stream, it's relative to your `detect.fps` setting.

**Comment by hawkeye217** [comment]
It's worth mentioning that setting your detect `fps` to anything higher than 5 (and at most 10) has little positive effect on most scenes and can slow down Frigate's object detection pipeline unnecessarily. The only time we suggest a value higher than 5 is if you're trying to track objects that move across the camera frame very quickly.

**Reply by edalquist** [reply]
Ack, I wasn't looking to increase detect fps, just doing the math in my head on "how long does something have to disappear for". I have a few cases where weather/rain seems to result in the object tracker "loosing" a parked car for a little bit and I wanted to try increasing `max_disappeared` to address it.

**Reply by hawkeye217** [reply]
Gotcha!
